frontend/src/pages/Vote/Landing.tsx

```diff
@@ -432,13 +551,56 @@ export default function Landing() {
           </RowBetween>
           {/* )} */}
 
+          <ThemedText.HeadlineLarge
+            style={{ margin: "28px 0 12px 0", flexShrink: 0 }}
+          >
+            <Trans>Top Delegates</Trans>
+          </ThemedText.HeadlineLarge>
+          <DelegateContainer>
+            <Delegate>
+              <DelegateAddress>Address</DelegateAddress>
+              <DelegateVotingContainer>
+                <DelegateVotingText>Voting Percentage</DelegateVotingText>
+                <DelegateVotingText>Voting Power</DelegateVotingText>
```

Comment:
Voting Power is expressed in % but the % not shown in the table. 
Can you add the % on the table in front of the number.